Adding -max_loading_error option and update to docs (#119)
tigergraph/gsql-docs: modules/ddl-and-loading/pages/running-a-loading-job.adoc

.`RUN LOADING JOB` syntax for concurrent loading
[source.wrap,ebnf]
----

> RUN LOADING JOB [-noprint] [-dryrun] [-n [i],j] [-max_percent_error [100] ] job_name [
>     USING file_var [="file_path_string"][, file_var [="file_path_string"]]*
>     [, CONCURRENCY=cnum][,BATCH_SIZE=bnum][,EOF="eof_mode"]
> ]

Reviewer: ```suggestion
RUN LOADING JOB [-noprint] [-dryrun] [-n [i],j] [-max_percent_error mpe ]
job_name [
    USING file_var [="file_path_string"]
         [, file_var [="file_path_string"]]*
    [, CONCURRENCY=cnum][,BATCH_SIZE=bnum][,EOF="eof_mode"]
]
```

Author: changed this with latest commit.

`-n 10, 50` means read lines 10 to 50.
The special symbol `$` is interpreted as "last line", so `-n 10,$` means reads from line 10 to the end.

> ==== `-max_percent_error`

Reviewer: ```suggestion
==== `-max_percent_error mpe`
```

Author: changed with new commit.

> The `-max_percent_error` option is used by the loader process to decide when to automatically abort a running loading job.
> This depends on a user provided percentage of the data input lines that are rejected.

Reviewer: ```suggestion
The `-max_percent_error` option instructs the loader process to abort the loading job if the percentage of data lines in the input batch that are rejected exceeds `mpe`.
For example, if the input source has 1000 data lines and `mpe` = 25, then the loading job will abort if it rejects 251 lines.
```

Author: changed this with latest commit.

There are two possible cases where an input line may be rejected:

> * Input format error: This is the case where the input line cannot be parsed properly.
> * Filtering error: This is the case where the input line can be parsed properly, but it does not pass certain filtering rules defined in the loading job itself.

Reviewer: ```suggestion
* Input format error: the input line cannot be parsed properly.
* Filtering error: the input line can be parsed properly, but it does not pass certain filtering rules defined in the loading job itself.
```

Author: changed this with latest commit.

For example, if `-max_percent_error` is set to 50%, and over 50% of data input lines in a given batch are rejected by the loader either due to parsing error or filtering error, the loader process will abort itself.

> NOTE: The default percentage is set to 0.

Reviewer 2: I recommend removing this line and instead mention the default behavior in the table

Author: removed with latest commit.

> For example, if `-max_percent_error` is set to 50%, and over 50% of data input lines in a given batch are rejected by the loader either due to parsing error or filtering error, the loader process will abort itself.
> 
> NOTE: The default percentage is set to 0.

Reviewer: ```suggestion
```
I suggested putting the example earlier, when `mpe` is defined.

Author: Changed with new commit.

> .`-max_percent_error` summery

Reviewer 2: Typo: summary

Reviewer: If you explained the meaning of -max_percent_error well and showed an example, then I think this whole table is unnecessary.  I could see this one example as helpful, because it represents a real use case:
- If mpe = 0, then the load job aborts if any input line is rejected.

(A user who wants 100 shouldn't even bother to set this option.

Author: Removed table an updated this with latest commit.

[cols="2"]
|===
|max_percent_error |behavior

> |0 |Loading will stop if any parsed input lines are rejected by the format or filtering rules.

Reviewer 2: In the document, we should specify the number range to be between 0 (non-inclusive) and 100 (inclusive).

Author: added motion of this in the latest commit.

|between 0 and 99 |Loading will stop if the percentage of rejected parsed input exceeds the user-provided limit.

> |100 |Loading will continue until complete.

Reviewer 2: Based on code logic "loading will continue until complete" is the default behavior

Author: Added a statement about default behavior and settings.
